virt/virtual_machines/virtual_disks/virt-deleting-datavolumes.adoc

```diff
@@ -6,12 +6,17 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can manually delete a data volume by using the `oc` command-line interface. +
-[NOTE]
-====
-When you delete a virtual machine, the data volume it uses is automatically deleted.
-====
+If you are running {VirtProductName} 4.12 and later, you do not need to manually delete a data volume, because the persistent volume claim (PVC) manages data volume deletion with an automated garbage collector.
+
+You can manually delete a data volume using an `oc` command. However, unexpected results can occur.
```

Comment:
This should probably be focused on deleting VM disks.  If that is the case then we can say something like the following.  To delete a VM disk: delete the associated DataVolume (if present).  If there is no DataVolume present it has already been garbage collected.  In this case, the VM disk can be removed by deleting the associated PVC directly.  If you delete a PVC that is owned by a DataVolume then CDI will recreate the PVC and attempt to import the disk again from the specified source.

Reply:
I can work with that. So it sounds like you are OK with leaving in the manual delete module. If so, I can go from there.